src/cwhy/cwhy.py

```diff
@@ -226,11 +217,6 @@ def __init__(self, diagnostic):
 
 def extract_sources_prompt(diagnostic):
     ctx = extract_sources_context(diagnostic)
```

Comment:
What happens now if stdin is empty? (Someone just types `cwhy` or the compiler outputs no errors...)

Reply:
I just factored out all the `stdin == ""` checks into main: `if stdin`. Behavior is unchanged.